hibernate-core/src/main/java/org/hibernate/dialect/CockroachDialect.java

```diff
@@ -95,56 +92,36 @@ public CockroachDialect(DatabaseVersion version, PostgreSQLDriverKind driverKind
 	}
 
 	@Override
-	protected List<Integer> getSupportedJdbcTypeCodes() {
-		List<Integer> typeCodes = new ArrayList<>( super.getSupportedJdbcTypeCodes() );
-		typeCodes.addAll( List.of(UUID, INTERVAL_SECOND, GEOMETRY, JSON) );
-		if ( getVersion().isSameOrAfter( 20 ) ) {
-			typeCodes.add(INET);
-		}
-		return typeCodes;
-	}
-
-	@Override
-	protected String columnType(int jdbcTypeCode) {
-		switch (jdbcTypeCode) {
+	protected String columnType(int sqlTypeCode) {
+		switch ( sqlTypeCode ) {
 			case TINYINT:
 				return "smallint"; //no tinyint
-
 			case CHAR:
 			case NCHAR:
 			case VARCHAR:
 			case NVARCHAR:
+			case LONGVARCHAR:
+			case LONGNVARCHAR:
 				return "string($l)";
-
-			case NCLOB:
 			case CLOB:
+			case NCLOB:
+			case LONG32VARCHAR:
+			case LONG32NVARCHAR:
```

Comment:
_Noooo_, please roll back changes like these.

These are *intentionally* not defined in every `Dialect`, and instead have sensible default interpretations defined elsewhere.

Please stop changing stuff like this.